### PR TITLE
Automatic banking counter

### PR DESCRIPTION
**Pull request by NathanaelZuchuon**

### CODE REVIEW

**Comment by NathanaelZuchuon** [comment]
How could I make it more shorter ?!

**Comment by estellemasso** [comment]
Keep up!

File: Automatic banking counter [hunk 1]
```diff
@@ -0,0 +1,196 @@
+class Guichet:
+
```
Comment (estellemasso) [inline Automatic banking counter:2]: Always add a description of the class here. Read https://realpython.com/python-comments-guide/

File: Automatic banking counter [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+	l_d = []
+	l_w = []
```
Comment (estellemasso) [inline Automatic banking counter:4]: Use more descriptive variables. How do I get to know what each of this does.

File: Automatic banking counter [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+	a = d = w = counter = 0
```
Comment (estellemasso) [inline Automatic banking counter:5]: Use more descriptive variables. How do I get to know what each of this does. Examples will be

File: Automatic banking counter (hunk 1 above)
Comment (estellemasso) [inline Automatic banking counter:2]: Always add a description of the class here. Read https://realpython.com/python-comments-guide/

File: Automatic banking counter [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+		self.limit = limit
+
+	def account_consultation(self):
+		(guichet.a)+=1
+		print("Vous avez %s ." % (self.amount))
+
+	def deposit(self):
+		new_amount = int(input('\nEntrer le montant de dépôt: '))
+		
+		if new_amount != 5:
```
Comment (estellemasso) [inline Automatic banking counter:20]: Why are you checking if new_amount != 5?

File: Automatic banking counter [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+	def withdraw(self):
+		new_amount = int(input('\nEntrer le montant de retrait: '))
+		
+		if new_amount != 5:
+			while (self.amount-new_amount)<(self.limit):
+				new_amount = int(input("Votre compte s'élèvera à %s, inférieur à la limite: recommencer... " % (self.amount-new_amount)))
+				if new_amount == 5:
+					guichet.cancel()
+					break
+			if (self.amount-new_amount)>=(self.limit):
```
Comment (estellemasso) [inline Automatic banking counter:37]: This if is no longer necessary

**Pull request merged**